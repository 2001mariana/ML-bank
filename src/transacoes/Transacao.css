.transacao-container {
    width: 32%;
    margin: 2px auto;
    display: grid;
    grid-template-columns: 1fr 3fr 1fr;
    background-color: #e6f1fe;
    padding: 0 10px;
}