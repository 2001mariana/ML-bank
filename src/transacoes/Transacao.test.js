import { render } from "@testing-library/react"
import Transacao from "./Transacao"
import React from 'react'

describe('Componente de transação de extrato', () => {
    it('snapshot do componente deve permanecer sempre o mesmo', 
        () => {
            const { container } = render(<Transacao
                data="09/09/2022"
                tipo="saque"
                valor="20.00"
            />)

            expect(container.firstChild).toMatchSnapshot();
        })
})