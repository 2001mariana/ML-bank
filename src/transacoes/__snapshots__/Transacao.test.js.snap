// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componente de transação de extrato snapshot do componente deve permanecer sempre o mesmo 1`] = `
<div
  class="transacao-container"
>
  <p>
    saque
  </p>
  <p>
    R$ 
    20.00
  </p>
  <p>
    09/09/2022
  </p>
</div>
`;
